hangman.py

```diff
@@ -113,21 +124,68 @@ def hangman(secret_word):
     
     Follows the other limitations detailed in the problem write-up.
     '''
-    # FILL IN YOUR CODE HERE AND DELETE "pass"
-    pass
-
+    print('''
+    __  __                                           ___ ____ 
+   / / / /___ _____  ____ _____ ___  ____ _____     <  // __ \\
+  / /_/ / __ `/ __ \/ __ `/ __ `__ \/ __ `/ __ \    / // / / /
+ / __  / /_/ / / / / /_/ / / / / / / /_/ / / / /   / // /_/ / 
+/_/ /_/\__,_/_/ /_/\__, /_/ /_/ /_/\__,_/_/ /_/   /_(_)____/  
+                  /____/                                      
+''')
+    # print(secret_word)
+    print(f'I am thinking of a word that is {len(set(secret_word))} letters long.')
+    guesses = 6
+    letters_guessed = []
+    vowels = ['a', 'e', 'i', 'o', 'u']
+    warnings = 3
+    while 1:
+        if is_word_guessed(secret_word, letters_guessed):
+            total_score = guesses * len(set(secret_word))
+            print(f'\033[7mCongratulations! You won, the secret word was - {secret_word}\033[0m')
+            print(f'\033[7mYour total score is {total_score}\033[0m')
+            break
+        elif guesses <= 0:
+            print(f'\033[7mYou lost. Guesses exceeded. The secret word was - {secret_word}\033[0m')
```

Comment:
I might be wrong, but probably there should be a different text

Reply:
Actually, where? I really dont get it.